test/metabase/driver/postgres_test.clj

```diff
@@ -264,9 +279,9 @@
                        ["DROP MATERIALIZED VIEW IF EXISTS test_mview;
                        CREATE MATERIALIZED VIEW test_mview AS
                        SELECT 'Toucans are the coolest type of bird.' AS true_facts;"])
-        (t2.with-temp/with-temp [Database database {:engine :postgres, :details (assoc details :dbname "materialized_views_test")}]
-          (is (= {:tables #{(default-table-result "test_mview")}}
-                 (driver/describe-database :postgres database))))))))
+        (mt/with-temp [:model/Database database {:engine :postgres, :details (assoc details :dbname "materialized_views_test")}]
+          (is (=? [(default-table-result "test_mview" {:estimated_row_count (mt/malli=? int?)})]
```

Comment:
Just for my understanding, why do you not have to execute `VACUUM` here but you do in the [other test](https://github.com/metabase/metabase/pull/39494/files#diff-821f0be02cebbea5f2f5b86239bd9df5e5cb4f671634a761fa6da1c511487125R1429)?

Reply:
for the `sync-estimated-row-count-test` test I want an accurate number, so executing VACUUM to trigger Postgres to get the exact number. but for these tests we don't need to test the exact number.

Reply:
got it, thanks